### ISSUE TITLE
cmake build problem

### ISSUE THREAD

**Issue by wsfulton** [jschueller/swig-field-example#1]
```
CMAKE_INSTALL_PREFIX=$PWD/install ..
-- Found SWIG: /home/william/swig/github/swig/swig (found version "4.0.0") 
CMake Error at CMakeLists.txt:34 (swig_add_library):
  Unknown CMake command "swig_add_library".


-- Configuring incomplete, errors occurred!
See also "/tmp/swig-field-example/build/CMakeFiles/CMakeOutput.log".
/tmp/swig-field-example/build (master=)$ cmake --version
cmake version 3.5.1
```
Does it need a newer version of CMake?

**Comment by jschueller** [comment]
Hi, it should be fixed by 9b9979f, can you give it another shot ?

**Comment by wsfulton** [comment]
Okay that works. Also add the following into the readme file after cmake has run:
```
make install
env PYTHONPATH=. python3 ../test-fieldfunction.py
```
